src/app/components/cds/alerts-and-prompts/Alert.js

```diff
@@ -106,8 +106,8 @@ Alert.defaultProps = {
 
 Alert.propTypes = {
   className: PropTypes.string,
-  title: PropTypes.object,
-  content: PropTypes.object,
+  title: PropTypes.oneOfType([PropTypes.object, PropTypes.string]),
```

Comment:
Fixes:
<img width="1027" alt="Screenshot 2023-10-12 at 3 48 34 PM" src="https://github.com/meedan/check-web/assets/418001/0916198f-7c17-4a3f-8a8d-0f9f0ab02a07">

Reply:
Perhaps this should be `PropTypes.node` instead?

Reply:
oh yeah, good call @amoedoamorim will update